src/main/webapp/app/course/tutorial-groups/shared/tutorial-group-detail/tutorial-group-detail.component.ts

```diff
@@ -48,6 +49,8 @@ export class TutorialGroupDetailComponent implements OnChanges {
                         if (change.currentValue && change.currentValue.tutorialGroupSessions) {
                             this.sessions = change.currentValue.tutorialGroupSessions;
                         }
+                        this.changeDetectorRef.detectChanges();
+                        break;
                     }
```

Comment:
Do you plan one adding more cases later on, otherwise the switch case and its break seams a bit off

Reply:
Removed switch in cffb9b45b8f3636fe3ddbd18e4e7f4f8528756e1